test/outreach/metadata/responses/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
We actually only want to return `id` plus the subfields within `attributes`, since those are the "real" fields of the object that hold a value. so the expected response is:

```
  "Result": {
    "mailings": {
      "DisplayName": "mailings",
      "FieldsMap": {
        "id": "id",
        "bodyHtml": "bodyHtml",
        "bodyText: "bodyText",
        "bouncedAt": "bouncedAt",
       // All the other fields in https://developers.outreach.io/api/reference/tag/Mailing/
      }
    },
   // ...
}
```